frontend/src/utils/filterHelpers.js

```diff
@@ -16,40 +16,38 @@ import { getFormatForDateField, getFormattedDate } from './widgetHelpers';
 export function formatFilters({ filtersData, filtersActive = [] }) {
   // for inline filters (if they were modified) in the response data we're getting a filter with
   // empty parameters
-  const filters = filtersActive
-    .filter((filter) => filter.parameters && filter.parameters.length)
-    .map((filter) => {
-      if (filter.parameters && filter.parameters.length) {
-        const filterData = getParentFilterFromFilterData({
-          filterId: filter.filterId,
-          filterData: filtersData,
-        });
+  const filters = filtersActive.map((filter) => {
+    const filterData = getParentFilterFromFilterData({
+      filterId: filter.filterId,
+      filterData: filtersData,
+    });
 
-        filter.parameters = filter.parameters.map((filterParameter) => {
-          const { parameterName, value, valueTo } = filterParameter;
-          const dataFilterParameter = filterData.parameters.find(
-            (param) => param.parameterName === parameterName
-          );
-          const { widgetType } = dataFilterParameter;
-
-          if (DATE_FIELD_TYPES.includes(widgetType)) {
-            const dateFormat = getFormatForDateField(widgetType);
-            const date = getFormattedDate(value, dateFormat);
-            const dateTo = getFormattedDate(valueTo, dateFormat);
-
-            filterParameter = {
-              parameterName,
-              value: date,
-              valueTo: dateTo,
-            };
-          }
+    if (filter.parameters && filter.parameters.length) {
+      filter.parameters = filter.parameters.map((filterParameter) => {
+        const { parameterName, value, valueTo } = filterParameter;
```

Comment:
scout boy rule: i think u can extract the body of `map` into a new function
e.g.
```
function formatFilterParameter(parameter) { ... }
```

Reply:
Well sortof. We have this `filterData` that is used in this map function, so in order to pass it there you have to create another anonymous function.